rollup.config.js

```diff
@@ -16,18 +16,23 @@ export default {
   },
   plugins: [
     json(),
-    babel({
-      exclude: 'node_modules/**',
-      plugins: ['@babel/plugin-proposal-object-rest-spread'],
-    }),
     resolve({
       browser: true,
     }),
     replace({
+      preventAssignment: false,
       BUILD_COMPAT_SNIPPET: 'true',
       BUILD_COMPAT_LOCAL_STORAGE: 'true',
       BUILD_COMPAT_2_0: 'true',
     }),
     commonjs(),
+    babel({
+      babelHelpers: 'runtime',
```

Comment:
do we want this to be 'runtime'? according to [this](https://github.com/rollup/plugins/tree/master/packages/babel#babelhelpers) there's some extra dependencies we need to include, and the old plugin behavior default was `bundled`, which is the same as experiment

Reply:
I tried 'bundled' initially, but I kept getting a build error: 
```
You have declared using "bundled" babelHelpers, but transforming index.ts resulted in "runtime"
```

I tried to work around this using other configurations but landed on this working for me first and fastest. The docs linked is what I initially was basing this change off of. You can see in [this commit](https://github.com/amplitude/Amplitude-JavaScript/pull/457/commits/67ceda934a0185f47e8c532711ce59fef6ab322e) I had to remove their suggested change make the tests pass since the tests relied on those global function definitions. 

I am not sold that this is the correct way to configure this. If I understand correctly, by defining this as runtime, these functions are not bundled with the package, so we are assuming that the package consumer will have these helper functions defined somewhere?

Reply:
I guess I could exclude the helpers for the esm and umd configs, and include them for tests?